src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -36,47 +37,62 @@ def check_estimand(self, estimand):
             return False
         return True
 
-    def check_input_columns(self, columns):
+    def check_input_columns(self, columns, election_id):
         """
         Check that input columns contain all neccessary values for a calculation
         """
-        missing_columns = [col for col in columns if col not in self.data_handler.data.columns]
+        missing_columns = []
+        if election_id == "G":
+            missing_columns = [col for col in columns if col not in self.data_handler.data.columns]
+        elif election_id == "P":
+            missing_columns = [
+                col for col in columns if col not in self.data_handler.data[self.data_handler.election_id]
+            ]
         return len(missing_columns) == 0
 
-    def predict_estimands(self, estimand):
+    def verify_estimand(self, estimand, election_id):
         """
-        Predict which estimands can be formed given a dataset and a list of estimands we would like to create
+        Verify which estimands can be formed given a dataset and a list of estimands we would like to create
         """
         if estimand not in self.transformation_map:
             raise ValueError(f"Estimand '{estimand}' is not supported.")
         self.transformations = self.transformation_map[estimand]
-        """
+
         if not self.check_input_columns(
-            [col for transform in self.transformations for col in transform.__code__.co_varnames[1:]]
+            [col for transform in self.transformations for col in transform.__code__.co_varnames[1:]], election_id
         ):
             return []
-        """
+
         return self.transformations
 
-    def create_estimand(self, estimand):
+    def create_estimand(self, estimand, given_function):
```

Comment:
```suggestion
    def create_estimand(self, estimand = None, given_function = None):
```
I think we want to default both of these to None? Or maybe just one of them.